staging/src/k8s.io/apiserver/pkg/storage/cacher/cache_watcher_test.go

```diff
@@ -650,3 +682,85 @@ func TestBookmarkAfterResourceVersionWatchers(t *testing.T) {
 		t.Fatalf("expected only one watcher to be expired")
 	}
 }
+
+func TestWatchListLatency(t *testing.T) {
```

Comment:
This test seems somewhat artificial to me - I'm not sure I see an actual value in it...